lib/honeybadger/notice.ex

```diff
@@ -51,6 +51,12 @@ defmodule Honeybadger.Notice do
       when is_map(metadata) and is_list(stacktrace) do
     new(exception_name, message, metadata, stacktrace, fingerprint)
   end
+
+  def new(%{class: exception_name}, metadata, stacktrace, fingerprint)
+      when is_map(metadata) and is_list(stacktrace) do
+    new(%{class: exception_name, message: ""}, metadata, stacktrace, fingerprint)
```

Comment:
@sorentwo I think it's better form to chain back to the "real map handler" over calling down to `new(exception_name, "", ...)` but change.

Reply:
Nevermind, just saw comment. Will remove.